lib/util/failover_uri.js

```diff
@@ -168,6 +168,9 @@ shaka.util.FailoverUri.prototype.createRequest_ =
         // effectively cache every response.
         this.requestPromise_ = null;
         this.request_ = null;
+	if(xhr.responseURL) {
```

Comment:
This whole section appears to be mis-indented. Please run the linter (./build/lint.sh) and compiler (./build/build.sh) and make sure that all checks pass before submitting a PR.